lib/board.rb

```diff
@@ -0,0 +1,46 @@
+class Board
+  attr_accessor :cells
+
+  @cells = []
+
+  def initialize
+    self.cells = Array.new(9, " ")
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.